kubernetes/monitoring/services/cmsmon-rucio-ds.yaml

```diff
@@ -19,26 +19,6 @@ spec:
       protocol: TCP
       targetPort: 31202
 ---
-kind: ConfigMap
```

Comment:
I suggest to keep config map instead of hard-coded parameters. All paths can be changed, include time of cronjob as well.